basic-dotnet-core/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,70 @@
+sb_application_name: basic-dotnet-core
+sb_build_namespace: basic-dotnet-core-build
+sb_dev_namespace: basic-dotnet-core-dev
+sb_stage_namespace: basic-dotnet-core-stage
+sb_prod_namespace: basic-dotnet-core-prod
+sb_application_repository_url: "https://github.com/redhat-developer/s2i-dotnetcore-ex.git"
```

Comment:
When testing I get an error here:

```
OpenShift Build basic-dotnet-core-build/basic-dotnet-core-pipeline-4 from https://github.com/redhat-cop/container-pipelines.git
Started by user josphill
Checking out git https://github.com/redhat-cop/container-pipelines.git into /var/lib/jenkins/jobs/basic-dotnet-core-build/jobs/basic-dotnet-core-build-basic-dotnet-core-pipeline/workspace@script to read basic-dotnet-core/Jenkinsfile
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/redhat-cop/container-pipelines.git # timeout=10
Fetching upstream changes from https://github.com/redhat-cop/container-pipelines.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/redhat-cop/container-pipelines.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/master^{commit} # timeout=10
Checking out Revision 1416d5f18537c261037260de14af902b1783141d (origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 1416d5f18537c261037260de14af902b1783141d
Commit message: "Merge pull request #95 from pabrahamsson/use_pipeline_library_v1_4"
 > git rev-list --no-walk 1416d5f18537c261037260de14af902b1783141d # timeout=10
ERROR: /var/lib/jenkins/jobs/basic-dotnet-core-build/jobs/basic-dotnet-core-build-basic-dotnet-core-pipeline/workspace@script/basic-dotnet-core/Jenkinsfile not found
Finished: FAILURE
```

Looks like that project/branch no longer has a Jenkinsfile?

Reply:
That would be because this pull request creates the jenkinsfile in the repo https://github.com/redhat-cop/container-pipelines.git and folder basic-dotnet-core, will need to manually override the variable sb_application_repository_url with my fork to test until this is merged

Reply:
I will update the testing directions above

Reply:
Updated the testing instructions above to override to use the fork